dynamic_rest/viewsets.py

```diff
@@ -337,4 +337,91 @@ def list_related(self, request, pk=None, field_name=None):
 
 
 class DynamicModelViewSet(WithDynamicViewSetMixin, viewsets.ModelViewSet):
-    pass
+
+    ENABLE_BULK_PARTIAL_CREATION = (
+        dynamic_settings.get('ENABLE_BULK_PARTIAL_CREATION', False))
```

Comment:
Adjusted this here: https://github.com/AltSchool/dynamic-rest/commit/da093dbc85d2d458d8f7e28f4d8ed4f25dcbf389